packages/calendar/react/src/components/calendar-item/calendar-item.tsx

```diff
@@ -218,23 +221,54 @@ ${extractEventHours(event)}`} // This is not a lint error. The space is left her
     events.forEach((event) => {
       const { id, title, startDate, endDate, account, attendees } = event;
 
-      const backgroundColor = account?.color.color;
-      const textColor = account?.color.darkColor;
+      // style the events
+      const fullConfig = resolveConfig({
+        ...tailwindConfig,
+        content: ["./src/**/*.{html,js,ts,tsx}"],
+      });
+
+      // FIXME: make typesafe
+      const eventColor = fullConfig.theme?.colors[account?.eventColor];
+      const textColorTw = fullConfig.theme?.textColor;
```

Comment:
we're doing this bit quite a few times, extract to some shared way?

fetch from trpc?

Reply:
Yes, should be extracted

Reply:
I haven't found a good way to extract the tailwind theme and make it available to `extension` but also `calendar-react`, which both need it.